nautobot/docs/user-guides/git-data-source.md

```diff
@@ -0,0 +1,273 @@
+![Example Details Jobs](./images/git-as-data-source/00-git-data-source_10.png)
+
+!!! tip
+    The same repository and branch can be used for the different `provides` methods.  Nautobot Git as a data source will look for specific root directory names.
+
+Once the scripts have been pushed into the repository, a sync needs to be executed, after which navigating to Jobs via **Extensibility -> Jobs** will show the new jobs loaded from the Git repository.
+
+![Default Repository Menu](./images/git-as-data-source/00-git-data-source_11.png)
+
+Jobs now shows the job from the Git repository.
+
+![List of Loaded Jobs](./images/git-as-data-source/00-git-data-source_12.png)
+
+At this point all changes, and history can be kept using Git.  A simple `sync` operation can be done from Nautobot to pulldown any changes.
+
+### Config Contexts
+Detailed information on [config contexts](../models/extras/gitrepository.md#configuration-contexts) in Git Repositories.
+
+Config contexts may be provided as JSON or YAML files located in `/config_contexts/`.
+
+Config contexts can be used to provide additional details to different automation tooling.  For example Ansible variables, or any other data that you can't natively store in Nautobot.  It can also be used in the Golden Configuration Nautobot plugin to provide extra details to generate configuration templates.
+
+A few simple examples of Configuration Context data might be:
+ - DNS Servers
+ - NTP Servers
+ - ACL Data
+ - Routing Information such as BGP ASNs etc.
+
+Similar to the other data sources, the repository can be added by navigating to **Extensibility -> Git repositories**. Click on **Add**, and fill out the repository details.
+
+![Example Details Config Contexts](./images/git-as-data-source/00-git-data-source_13.png)
+
+Once the repository syncs the details can be found in the **Synchronization Status** tab.  For example the platform specifics were synced.
+
+![Syncronization Menu With Loaded Contexts](./images/git-as-data-source/00-git-data-source_14.png)
+
+The repository structure is:
+```no-highlight
+▶ tree config_contexts 
+config_contexts
+├── devices
+│   ├── site-a-bb-01.yml
+│   ├── site-a-rtr-01.yml
+│   ├── site-a-rtr-02.yml
+│   ├── site-a-spine-01.yml
+│   ├── site-a-spine-02.yml
+│   ├── site-b-bb-01.yml
+│   ├── site-b-leaf-01.yml
+│   ├── site-b-leaf-02.yml
+│   ├── site-b-rtr-01.yml
+│   ├── site-b-rtr-02.yml
+│   ├── site-b-spine-01.yml
+│   └── site-b-spine-02.yml
+├── platform_eos.yml
+├── platform_junos.yml
+├── platform_nxos.yml
+└── role_spine.yml
+
+1 directory, 16 files
+```
+
+Configuration Context details:
+- Follows an inheritance methodology similar to what Ansible implements.  Global contexts can be overwritten by local contexts at both a group level, as well as at a device specific level.
+- Nautobot UI provides a simple view to see merged config contexts.  It can be visualized by navigating to a device and clicking on the **config contexts** tab.
```

Comment:
Lists not rendering properly here, too:
```suggestion
Configuration Context details:

- Follows an inheritance methodology similar to what Ansible implements.  Global contexts can be overwritten by local contexts at both a group level, as well as at a device specific level.
- Nautobot UI provides a simple view to see merged config contexts.  It can be visualized by navigating to a device and clicking on the **config contexts** tab.
```